src/components/PageBanner/index.js

```diff
@@ -15,7 +15,7 @@ const PageBanner = ({ imagesPath, imageFilename, imageExtension, imageTitle, hea
       altText={imageTitle}
     />
     <div
-      className="coa-PageBannerCover"
+      className={`coa-PageBannerCover ${extraClasses ? extraClasses : ''}`}
       role="img"
       aria-label={imageTitle}
     >
```

Comment:
Here we allow the banner (image) component to have additional classes if needed.